core/primitives/src/genesis/block.rs

```diff
@@ -67,14 +67,7 @@ impl Block {
         }
         let vrf_value = near_crypto::vrf::Value([0; 32]);
         let vrf_proof = near_crypto::vrf::Proof([0; 64]);
-        let body = BlockBody::new(
-            genesis_protocol_version,
-            chunks,
-            challenges,
-            vrf_value,
-            vrf_proof,
-            chunk_endorsements,
-        );
+        let body = BlockBody::new(chunks, challenges, vrf_value, vrf_proof, chunk_endorsements);
```

Comment:
This is where it's useful to have the code separated out for `latest_genesis` and `prod_genesis` :)